### PR TITLE
Tgbot 42 Create object provider for Car objects

### PR DESCRIPTION
**Pull request by Sav4ykVladzislau**
https://schneidervs.atlassian.net/browse/TGBOT-42

### CODE REVIEW

File: stoservice/src/main/java/com/tutrit/stoservice/repository/CarRepository.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public Car updateCar(Car car) {
     }
 
     public boolean deleteCar(Car car) {
+
```
Comment (yurasava) [inline stoservice/src/main/java/com/tutrit/stoservice/repository/CarRepository.java:49]: It's not needed

File: stoservice/src/main/java/com/tutrit/stoservice/bean/Car.java [hunk 2]
```diff
@@ -10,12 +10,12 @@ public class Car {
     private String generation;
     private String modification;
     private String engine;
-    private int year;
+    private String year;
```
Comment (yurasava) [inline stoservice/src/main/java/com/tutrit/stoservice/bean/Car.java:13]: Why did you change the type of the variable?

**Review by yurasava — APPROVED** [review]
Look, your pull request contains Vova commits.

**Comment by schneidervs** [comment]
need to redo the branch and pull request.

File: stoservice/src/main/java/com/tutrit/stoservice/bean/Car.java (hunk 2 above)
Comment (schneidervs) [reply stoservice/src/main/java/com/tutrit/stoservice/bean/Car.java:13]: I wouldn't change

File: stoservice/src/main/java/com/tutrit/stoservice/repository/CustomerRepository.java [hunk 3]
```diff
@@ -4,73 +4,70 @@
...
-    public Customer createCustomer(Customer customer) {
-        if (customers.contains(customer) == false) {
-            customers.add(customer);
-        } else {
-            logger.info("Customers already exists");
+        public Customer createCustomer(Customer customer) {
+            if (customers.contains(customer) == false) {
+                customers.add(customer);
+            } else {
+                logger.info("Customers already exists");
```
Comment (schneidervs) [inline stoservice/src/main/java/com/tutrit/stoservice/repository/CustomerRepository.java:18]: Can if not added return NULL

File: stoservice/src/main/java/com/tutrit/stoservice/repository/CustomerRepository.java [hunk 4]
```diff
@@ -4,73 +4,70 @@
...
+            return null;
         }
-        return null;
-    }
 
-    public Customer updateCustomer(Customer customer) {
-        if (customers.contains(customer) == true) {
+        public Customer updateCustomer(Customer customer) {
+            if (customers.contains(customer) == true) {
+                return customer;
```
Comment (schneidervs) [inline stoservice/src/main/java/com/tutrit/stoservice/repository/CustomerRepository.java:43]: He should update in the repository

**Review by schneidervs — APPROVED** [review]

**Comment by schneidervs** [comment]
Probably it is not necessary to mention Order Repository in this request

**Review by AlexanderYarokhovich — APPROVED** [review]
good
